code/modules/paperwork/faxmachine.dm

```diff
@@ -169,9 +181,19 @@ var/list/sent_faxes = list()	//cache for faxes that have been sent by the admins
 
 	updateUsrDialog()
 
-/obj/machinery/photocopier/faxmachine/proc/sendfax(var/destination)
+// Only run this in a spawn()!
+/obj/machinery/photocopier/faxmachine/proc/process_cooldown()
```

Comment:
This should be worked into the fax machine's process() proc instead. delay(), spawn(), or sleep() shouldn't be used in a process-like proc, as they remove control over scheduling from the actual ingame controllers.

You'll need to save the world.time of last process, deduct that from current process to get delta time, and then deduct delta time from sendcooldown. Also make sure that `. = ..()` is in there.